tpl/bootstrap5.php

```diff
@@ -353,7 +353,7 @@
 <?php
 endif;
 ?>
-				<div id="status" role="alert" class="alert alert-info<?php echo empty($STATUS) ? ' hidden' : '' ?>">
+				<div id="status" role="alert" class="alert alert-<?php $STATUS == 'Paste was properly deleted.' ? 'success' : 'info' ?><?php echo empty($STATUS) ? ' hidden' : '' ?>">
```

Comment:
Same reasoning as above: I wouldn't change the `#status` box design for this case, as it will then stay in `alert-success` (green) mode even when other informational (blue) messages get displayed.

Reply:
Same justification as above.